### ISSUE TITLE
source-map-resolve incompatibility

### ISSUE THREAD

**Issue by jkassis** [maxdavidson/rollup-plugin-sourcemaps#120]
`
(node:49685) ExperimentalWarning: The ESM module loader is experimental.
file:///Users/jkassis/Code/merchie/server/node_modules/rollup-plugin-sourcemaps/dist/index.js:4
import { resolveSourceMap, resolveSources } from 'source-map-resolve';
                           ^^^^^^^^^^^^^^
SyntaxError: The requested module 'source-map-resolve' does not provide an export named 'resolveSources'
    at ModuleJob._instantiate (internal/modules/esm/module_job.js:90:9)
    at async ModuleJob.run (internal/modules/esm/module_job.js:107:20)
    at async Loader.import (internal/modules/esm/loader.js:179:24)
module_job.js:90
Waiting for the debugger to disconnect...

`

**Comment by jkassis** [comment]
i forked this, made no changes, did my own build with gitpkg and it seems to work... shrug.

`jkassis/rollup-plugin-sourcemaps#rollup-plugin-sourcemaps-v0.6.2-gitpkg`

**Comment by Silic0nS0ldier** [comment]
The issue here is named imports being used with `source-map-resolve` which is a CommonJS package. https://nodejs.org/api/esm.html#esm_import_statements

`master` has the fix for this (https://github.com/maxdavidson/rollup-plugin-sourcemaps/compare/v0.6.2...master), just needs to be released. Anything blocking this that can be helped with @maxdavidson  ?

**Comment by Silic0nS0ldier** [comment]
Workaround in the meantime.

```js
import { createRequire } from 'module';
const require = createRequire(import.meta.url);
const sourcemaps = require("rollup-plugin-sourcemaps");
```

**Comment by maxdavidson** [comment]
I'm sorry for the delay, but this should now be fixed in [v0.6.3](https://github.com/maxdavidson/rollup-plugin-sourcemaps/tree/v0.6.3)
